general-configuration-guide/use-tinymce-distraction-free.md

```diff
@@ -75,12 +77,14 @@ tinymce.init({
 
 ### Related configuration options
 
-* [quickbars_insert_toolbar]({{ site.baseurl }}/configure/editor-appearance/#quickbars_insert_toolbar)
-* [quickbars_selection_toolbar]({{ site.baseurl }}/configure/editor-appearance/#quickbars_selection_toolbar)
-* [inline]({{ site.baseurl }}/configure/editor-appearance/#inline)
-* [skin]({{ site.baseurl }}/configure/editor-appearance/#skin)
-* [theme]({{ site.baseurl }}/configure/editor-appearance/#theme)
-
+For information on:
+* Configuring a Quick Toolbar; including `quicklink`, `quickimage`, and `quicktable`; see: [Quick Toolbar plugin]({{site.baseurl}}/plugins/quickbars/).
+* Configuring a contextual toolbar, see: [Context Toolbar]({{site.baseurl}}/ui-components/contexttoolbar/).
+* Configuring a contextual menu, see: [Context Menu]({{site.baseurl}}/ui-components/contextmenu/).
+* Disabling the {{site.productname}} menubar, see: [Menubar]({{site.baseurl}}/configure/editor-appearance/#menubar).
+* The `quickbars_insert_toolbar` setting, see: [quickbars_insert_toolbar]({{site.baseurl}}/configure/editor-appearance/#quickbars_insert_toolbar/).
+* The `quickbars_selection_toolbar` setting, see: [quickbars_selection_toolbar]({{ site.baseurl }}/configure/editor-appearance/#quickbars_selection_toolbar).
```

Comment:
yeah, I spotted that too. I think I'll fix this up when I start revising the doc set (it will break links, etc.) I just want to remove the strange and incomplete list at the bottom of this page for now.

Reply:
scratch that. Including the setting info on the quickbar plugin and deleted these items